src/displayapp/screens/settings/SettingSetTime.cpp

```diff
@@ -52,7 +52,7 @@ SettingSetTime::SettingSetTime(Pinetime::Applications::DisplayApp* app,
     hourCounter.EnableTwelveHourMode();
   }
   hourCounter.SetValue(dateTimeController.Hours());
-  lv_obj_align(hourCounter.GetObject(), nullptr, LV_ALIGN_CENTER, -75, POS_Y_TEXT);
+  lv_obj_align(hourCounter.GetObject(), nullptr, LV_ALIGN_CENTER, -72, POS_Y_TEXT);
```

Comment:
I think this was an unintentional change.

Reply:
![Screenshot 2023-02-05 163641 (Custom) (1)](https://user-images.githubusercontent.com/88860704/216847706-caec903f-d3ae-42f5-9c51-e44b5ebc36a2.jpg)
it aligns the counter better with the button (left is new change)

Reply:
This is not an improvement. The button width should be adjusted instead.

![Untitled](https://user-images.githubusercontent.com/37774658/216892458-abe81f1e-2b57-4a8f-b3ff-7c11232db941.png)